libraries/AP_AHRS/AP_AHRS_DCM.cpp

```diff
@@ -277,19 +274,16 @@ AP_AHRS_DCM::renorm(Vector3f const &a, Vector3f &result)
  *  to approximations rather than identities. In effect, the axes in the two frames of reference no
  *  longer describe a rigid body. Fortunately, numerical error accumulates very slowly, so it is a
  *  simple matter to stay ahead of it.
- *  We call the process of enforcing the orthogonality conditions �renormalization�.
+ *  We call the process of enforcing the orthogonality conditions �renormalization�.
```

Comment:
Both the original and the new characters appear strange on the github webpage. Are these supposed to be simple quotes ?